### ISSUE TITLE
META.yml lists non-existent module t::Util as prerequisite

### ISSUE THREAD

**Issue by eserte** [karupanerura/p5-Filesys-Notify-KQueue#2]
While installing the Filesys-Notify-KQueue-0.10 with CPAN.pm:

```
...
Running test for module 't::Util'

  The module t::Util isn't available on CPAN.

  Either the module has not yet been uploaded to CPAN, or it is
  temporary unavailable. Please contact the author to find out
  more about the status. Try 'i t::Util'.
  KARUPA/Filesys-Notify-KQueue-0.10.tar.gz
...
```

**Comment by karupanerura** [comment]
oops!

**Comment by karupanerura** [comment]
I released it as version 0.11. thank you for reporting!
